OpenRA.Mods.Common/Widgets/HotkeyEntryWidget.cs

```diff
@@ -86,7 +87,8 @@ public override bool HandleMouseInput(MouseInput mi)
 			Keycode.RSHIFT, Keycode.LSHIFT,
 			Keycode.RCTRL, Keycode.LCTRL,
 			Keycode.RALT, Keycode.LALT,
-			Keycode.RGUI, Keycode.LGUI
+			Keycode.RGUI, Keycode.LGUI,
+			Keycode.RETURN, Keycode.KP_ENTER
```

Comment:
Moving these here will change `HandleKeyPress` to return false and not call `YieldKeyboardFocus`. Have you considered the implications for this?

Reply:
The only time when the user can press <kbd>Enter</kbd> is when the hotkey is invalid. And since losing focus triggers hotkey validation the field just goes back to the focused state. That's why I added the return/enter keys to the list of ignored keys - to spare that round-trip through the logic. 

I thought about wiring <kbd>Enter</kbd> to the override action since that key is usually used to accept things (and is the opposite of <kbd>Esc</kbd>). But perhaps it isn't too obvious in this case that pressing enter would result in an override and it may be an unexpected action.

Reply:
Ah, my bad. I somehow missed that this was `HotkeyEntryWidget` and not the more general `TextFieldWidget`.